packages/forms-web-app/src/views/full-appeal/submit-final-comment/final-comment.njk

```diff
@@ -49,7 +49,7 @@
           name: "does-not-include-sensitive-information",
           items: [
             {
-              checked: (hasSensitiveInformation === false),
```

Comment:
removed this as otherwise the checkbox state is hardcoded - hasSensitiveInformation state is now passed from controller